api/src/util.ts

```diff
@@ -0,0 +1,9 @@
+import prefixes from '../../cdk/src/open-data-platform/frontend/url-prefixes';
+
+export const CORS_HEADERS = {
+  'Access-Control-Allow-Headers': 'Content-Type',
+  'Access-Control-Allow-Origin': '*',
```

Comment:
Seems like a very permissive CORS origin config, any reason not to dynamically set CORS based on the apps domain?

Reply:
I added a TODO to lock this down or remove the CORS headers altogether, since that might be handled at the gateway layer instead.